tests/test_resources.py

```diff
@@ -22,6 +22,12 @@ def test_init_raises_on_invalid_mem(self):
         with pytest.raises(ValueError):
             Resources(memory_limit="4")
 
+    def test_init_raises_on_invalid_eph(self):
+        with pytest.raises(ValueError):
+            Resources(ephemeral_request="3m")
+        with pytest.raises(ValueError):
+            Resources(ephemeral_limit="2m")
```

Comment:
@lancour looks like hera uses `assert`s, so these need to be `AssertionError`:
```suggestion
        with pytest.raises(AssertionError):
            Resources(ephemeral_request="3m")
        with pytest.raises(AssertionError):
            Resources(ephemeral_limit="2m")
```